newsApi.js

```diff
@@ -1,5 +1,12 @@
-// class NewsApi {
+class NewsApi {
+  constructor() {
+    this.url = `https://content.guardianapis.com/search?order-by=newest&show-fields=thumbnail&api-key=${process.env.API_KEY}`
```

Comment:
Assigning the URL to a variable within your constructor is a very neat solution. Looks good!